pygeoapi/provider/mvt.py

```diff
@@ -104,7 +110,28 @@ def get_layer(self):
 
         if is_url(self.data):
             url = urlparse(self.data)
-            return url.path.split("/{z}/{x}/{y}")[0][1:]
+            # We need to try, at least these different variations that
+            # I have seen across products (maybe there more??)
+
+            if ("/{z}/{x}/{y}" not in url.path and
```

Comment:
Check if people are using a different order for z,x,y than the ones we support. If there are use cases for this, we can address them later. For performance reasons, I wanted to avoid checking every combination in the code, if it is not required...